package-lock.json

```diff
@@ -1,13275 +1,8 @@
 {
   "name": "ibm-continuous-delivery",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
The package-lock.json changes seem unnecessary since there's no change to the packages in package.json. Can you back this file out?